src/exercise/01.js

```diff
@@ -3,27 +3,25 @@
 
 import * as React from 'react'
 
-function Greeting() {
-  // 💣 delete this variable declaration and replace it with a React.useState call
-  const name = ''
+function Greeting({initialName = ''}) {
+  const [name,setName] = React.useState(initialName)
 
   function handleChange(event) {
-    // 🐨 update the name here based on event.target.value
+    setName(event.target.value)
   }
-
   return (
     <div>
       <form>
         <label htmlFor="name">Name: </label>
-        <input onChange={handleChange} id="name" />
+        <input value={name} onChange={handleChange} id="name" />
```

Comment:
`value` should not be needed here; interesting twist....

Reply:
Yeah he had that in his to prepopulate the field... seemed nice so I kept it haha